sql/core/src/test/resources/sql-tests/results/describe.sql.out

```diff
@@ -890,6 +890,48 @@ a                   	string              	CONCAT('a\n b\n ', 'c\n d')
 b                   	int                 	42
 
 
+-- !query
+CREATE TABLE f PARTITIONED BY (B, C) AS SELECT 'APACHE' A, CAST('SPARK' AS BINARY) B, TIMESTAMP'2018-11-17 13:33:33' C
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+DESC FORMATTED f PARTITION (B='SPARK', C=TIMESTAMP'2018-11-17 13:33:33')
+-- !query schema
+struct<col_name:string,data_type:string,comment:string>
+-- !query output
+A                   	string              	                    
+B                   	binary              	                    
+C                   	timestamp           	                    
+# Partition Information	                    	                    
+# col_name          	data_type           	comment             
+B                   	binary              	                    
+C                   	timestamp           	                    
+                    	                    	                    
+# Detailed Partition Information	                    	                    
+Database            	default             	                    
+Table               	f                   	                    
+Partition Values    	[B=SPARK, C=2018-11-17 13:33:33]	                    
+Location [not included in comparison]/{warehouse_dir}/f/B=SPARK/C=2018-11-17 13%3A33%3A33	                    
+Partition Parameters	[numFiles=1, totalSize=486, transient_lastDdlTime=[not included in comparison]]	                    
```

Comment:
we have `transient_lastDdlTime` because it's a hive table?

Reply:
Maybe we can test with `... AS JSON` to avoid seeing these unrelated fields.

Reply:
> we have transient_lastDdlTime because it's a hive table?

This is not from table parameters, but partition stats, I'm not aware of the history of filtering out some of the hive properties. However, it's not related to this PR.

> Maybe we can test with ... AS JSON to avoid seeing these unrelated fields.

We already have an `AS JSON` test variant, unfortunately transient_lastDdlTime is still there

Reply:
well, at least the masking logic is already implemented for AS JSON and we can save some changes from this PR :)

Reply:
This PR also added the masking for transient_lastDdlTime, do we have any concerns?